src/lib/constants.ts

```diff
@@ -70,6 +70,7 @@ export const KCL_DEFAULT_COLOR = `#3c73ff`
 export const SETTINGS_FILE_NAME = 'settings.toml'
 export const PROJECT_SETTINGS_FILE_NAME = 'project.toml'
 export const COOKIE_NAME = '__Secure-next-auth.session-token'
+export const COOKIE_NAME_DEV = '__Secure-session-token-dev'
```

Comment:
I know this is just a draft, but I don't want to see a literal "dev" propagate any further in this codebase. The term is far too overloaded. 😄

Reply:
on zoo.dev and gov we can not use the new names so I had to make an exception for it.

But the name "dev" does not longer show up in this patch.